package.json

```diff
@@ -39,7 +39,7 @@
     "fast-deep-equal": "^3.1.3",
     "lodash.get": "^4.4.2",
     "lodash.set": "^4.3.2",
-    "private-group-spec": "^3.0.0",
+    "private-group-spec": "github:ssbc/private-group-spec#groupkey-in-init",
```

Comment:
https://github.com/ssbc/private-group-spec/pull/25

Reply:
After merging that one, and publishing, then we can merge this tribes2 PR